content/drone-plugins/drone-s3-cache/index.md

```diff
@@ -0,0 +1,174 @@
+---
+date: 2016-01-01T00:00:00+00:00
+title: Amazon S3 Cache
+author: drone-plugins
+tags: [ amazon, aws, s3, cache ]
+repo: drone-plugins/drone-s3-cache
+logo: s3_cache.svg
+image: plugins/s3-cache
+---
+
+The S3 cache plugin can be used to preserve files and directories between builds. The below pipeline configuration demonstrates simple usage:
+
+```yaml
+pipeline:
+  restore-cache:
+    image: plugins/s3-cache:1
```

Comment:
we don't need to specify the version tag

Reply:
@bradrydzewski and me just talked about it that it would make sense to use specific version tags, so I just started to do it.

Reply:
> Brad Rydzewski @bradrydzewski Aug. 28 21:59
> @tboerger one interesting thing we could do (not sure if it is a good idea or not) is to not publish latest tags for images and force people to always use a version number (ie plugins/slack:1) or at least maybe we always document plugin usage with versions
>
> Thomas Boerger @tboerger Aug. 28 22:00
> i would keep latest but write on the examples or docs to use a version to stay compliant 
>
> Brad Rydzewski @bradrydzewski Aug. 28 22:00
> yea, I think that makes more sense